app/src/main/java/app/grapheneos/camera/capturer/ImageCapturer.kt

```diff
@@ -32,22 +31,27 @@ class ImageCapturer(val mActivity: MainActivity) {
     var cFadeAnim : AlphaAnimation? = null
 
     private fun fadeCaptureButton() {
-        mActivity.captureButton.isEnabled = false
 
+        val captureButton = mActivity.currentCaptureButton
+
+        captureButton.isEnabled = false
         cFadeAnim?.cancel()
 
-        AlphaAnimation(mActivity.captureButton.alpha, 0.6f).let {
+        AlphaAnimation(captureButton.alpha, 0.6f).let {
             it.duration = 200
             it.interpolator = LinearInterpolator()
             it.fillAfter = true
 
-            mActivity.captureButton.startAnimation(it)
+            captureButton.startAnimation(it)
             cFadeAnim = it
         }
     }
 
     private fun unfadeCaptureButton() {
-        mActivity.captureButton.isEnabled = true
+
+        val captureButton = mActivity.currentCaptureButton
+
+        captureButton.isEnabled = true
```

Comment:
Lines 58 and 63 need to be updated too.

Reply:
I think there's probably some GitHub UI issue here. Could you please clarify on this review once?
Thanks a lot for your valuable time and feedback @muhomorr!

Reply:
I mean lines 58 and 63 still refer to `mActivity.captureButton` (not the same as `captureButton`)

Reply:
See changes on lines 40 and 45

Reply:
I'm really sorry but am unable to get you here. We're loading the current capture button to a local `captureButton` variable so we're ideally always referring to the instance returned by the `currentCaptureButton` getter field for a given method invocation.

Reply:
> We're setting the current capture button to local variable captureButton, so we're ideally always referring to the instance returned by the currentCaptureButton getter field for a given method invocation.

Yes, this is correct. And `fadeCaptureButton()` was correctly updated to use `captureButton` local variable.
But `unfadeCaptureButton()` wasn't fully updated.
Line 54 correctly refers to `captureButton` local variable.
But lines
58 `AlphaAnimation(mActivity.captureButton.alpha, 1f).let {`
and 
63 `mActivity.captureButton.startAnimation(it)`

refer to `mActivity.captureButton` variable which is different from `captureButton` local variable.

Reply:
Got it. Thanks for the feedback @muhomorr!